relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -293,10 +350,9 @@ fn scrub_resource(resource_type: &str, string: &str) -> Option<String> {
             return Some("browser-extension://*".to_owned());
         }
         scheme => {
-            let domain = url
-                .domain()
-                .and_then(|d| normalize_domain(d, url.port()))
-                .unwrap_or("".into());
+            let scrubbed_host = url.host().map(scrub_host).unwrap_or(String::from(""));
```

Comment:
With the comments above, if `scrub_ipv4` becomes a `Cow`, `scrub_host` returns a `Cow` => can also borrow here.
```suggestion
            let scrubbed_host = url.host().map(scrub_host).unwrap_or(Cow::Borrowed(""));
```